File: app/src/main/java/com/vego/vego/APIs/FirebaseAPI.java
package com.vego.vego.APIs;

import java.util.HashMap;

import retrofit2.Call;
import retrofit2.http.Body;
import retrofit2.http.Header;
import retrofit2.http.POST;

public interface FirebaseAPI {
    @POST("fcm/send")
    Call<okhttp3.ResponseBody> sendNotifi(@Body HashMap<String, Object> body,
                                                 @Header("Authorization") String authorization);
}

File: app/src/main/java/com/vego/vego/APIs/FirebaseAPI.kt
package com.vego.vego.APIs

import okhttp3.ResponseBody
import org.json.simple.JSONObject
import retrofit2.Call
import retrofit2.http.Body
import retrofit2.http.Header
import retrofit2.http.POST

interface FirebaseAPI {
    @POST("fcm/send")
    fun sendNotifi(@Body body: org.json.simple.JSONObject, @Header("Authorization") authorization: String): Call<ResponseBody>
}